packages/core/src/providers/yandex.ts

```diff
@@ -0,0 +1,58 @@
+import { OAuthConfig, OAuthUserConfig } from "."
+
+export interface YandexProfile {
+  id: string
+  login: string | undefined
+  client_id: string
+  display_name: string
+  real_name: string
+  first_name: string
+  last_name: string | undefined
+  sex: string | undefined
+  default_email: string | undefined
+  emails: string[]
+  psuid: string
+  // bithday will be in YYYY-MM-DD formated string if exists
+  birthday: string | undefined
+  is_avatar_empty: boolean | undefined
+  default_avatar_id: string | undefined
+  default_phone: { id: number; number: string } | undefined
+}
+
+export default function Yandex(
```

Comment:
Can we document this function? See https://github.com/nextauthjs/next-auth/blob/main/packages/core/src/providers/github.ts for an example.